examples/breakout-rooms/src/components/BreakoutManager/BreakoutManager.tsx

```diff
@@ -239,10 +245,10 @@ export const BreakoutManager: React.FC = () => {
 
       {isEditable && validations.length > 0 ? (
         <div className="validation-messages">
-          <h5 className="validatino-messages-title">
+          <h5 className="validation-messages-title">
             Before starting the session:
           </h5>
-          <ul className="validatino-messages-items">
+          <ul className="validation-messages-items">
             {validations.map((message) => (
               <li key={message}>{message}</li>
```

Comment:
Can you add a class so we dont need the `.validation-messages-items li + li` selector?

Reply:
Done @ 6f6bb6d135dff6586656932d093ecbacd259cee2